### PR TITLE
install cve fix

### PR DESCRIPTION
**Pull request by mfleader**
## Changes introduced with this PR

Install CVE fix for `requests`.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by jaredoconnell — APPROVED** [review]

File: Dockerfile.buildbase [hunk 1]
```diff
@@ -7,6 +7,9 @@ RUN dnf -y install gcc-c++ python3-devel openssl-devel
 RUN python3.9 -m pip install poetry==1.4.2 \
  && python3.9 -m poetry config virtualenvs.create false
 
+# Poetry 1.4.2 requires requests = "^2.18"
+# We need requests = "^2.32.0" to fix cve-2024-35195
+RUN python3.9 -m pip install requests="^2.32.0"
```
Comment (webbnh) [inline Dockerfile.buildbase]: Perhaps it's no big deal, but it looks to me like we are actually _running_ Poetry at line 8 _**before**_ updating the `requests` package at line 12...which sort of creates a small hole....

**Review by webbnh — COMMENTED** [review]
A nugget for your consideration....

File: Dockerfile.buildbase (hunk 1 above)
Comment (mfleader) [reply Dockerfile.buildbase]: sure why not

**Review by webbnh — APPROVED** [review]
👍

**Pull request merged**